qualang_tools/results/results.py

```diff
@@ -44,7 +48,14 @@ def is_processing(self):
 
         :return: boolean flag which is True while the program is processing.
         """

```

Comment:
We also need to update the docstring.
Maybe we also need to rename the function?

Reply:
I don't know, because it does the same thing as before just more efficiently, why do you want to rename it ?

Reply:
It's not 100% representative, because it also returns true the first time it is called after it is done processing.
But I'm ok with the name, just need to add it to the docstring :)